futures-util/src/stream/mod.rs

```diff
@@ -868,6 +868,9 @@ pub trait StreamExt: Stream {
     /// This combinator will attempt to pull items from both streams. Each
     /// stream will be polled in a round-robin fashion, and whenever a stream is
     /// ready to yield an item that item is yielded.
+    ///
+    /// When either this stream or the provided one ends, the remaining stream will be polled to
+    /// completion or error. The stream that ended will no longer be polled.
```

Comment:
Streams won't necessarily stop being polled on error, especially since this isn't a method on `TryStream`, which has special-casing for error handling. I would leave out the "or error" bit.

Reply:
Too used to futures 0.1. Will remove the or error part.

Reply:
Thanks!  :smile: